activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/rest/validation/GenericModelPayloadValidator.java

```diff
@@ -45,7 +51,7 @@ public boolean supports(Class<?> clazz) {
     public void validate(Object target,
                          Errors errors) {
         Model model = (Model) target;
-        if (checkRequiredFields || model.getName() != null) {
+        if (model.getName() != null) {
```

Comment:
checkRequiredFields is for the case when you update the model but don't specify the name: in this case the validation should pass because we will keep the existing name.

Reply:
We need to discuss this, because also this produces a 500 Error when you want to create a model but the field name is null -> You will get 500.